### ISSUE TITLE
Adding/Editing stream host causes: data.forwarding_host should match exactly one schema in oneOf

### ISSUE THREAD

**Issue by mgutt** [jc21/nginx-proxy-manager#1375]
I was not able to find a similar error. Screenshot:
![image](https://user-images.githubusercontent.com/10757176/132100966-659b9233-3152-4ae6-bdff-8010552ba31e.png)

"schema" sounds like a database problem?! I'm using /data/database.sqlite through the DB_SQLITE_FILE variable.

Container is created as follows:

```
docker run -d --name='npm' --net='host' -e TZ="Europe/Berlin" -e HOST_OS="Unraid" -e 'TCP_PORT_81'='81' -e 'TCP_PORT_80'='80' -e 'TCP_PORT_443'='443' -e 'DB_SQLITE_FILE'='/data/database.sqlite' -e 'TCP_PORT_25565'='25565' -e 'TCP_PORT_25566'='25566' -v '/mnt/cache/appdata/Nginx-Proxy-Manager-Official/data':'/data':'rw' -v '/mnt/cache/appdata/Nginx-Proxy-Manager-Official/letsencrypt':'/etc/letsencrypt':'rw' --memory=1G 'jc21/nginx-proxy-manager'
```

HTTP Request is sent as POST to /api/nginx/streams but a 400 Bad Request error is returned. Post content:
```
{"incoming_port":25576,"forwarding_host":"192.168.178.8","forwarding_port":25566,"tcp_forwarding":true,"udp_forwarding":false}
```

Is this error returned by Nginx? I found only this in /app/routes/api/nginx/streams.js, which seem to listen to the POST request:
```
/**
 * /api/nginx/streams
 */
router
        .route('/')
        .options((req, res) => {
                res.sendStatus(204);
        })
        .all(jwtdecode()) // preferred so it doesn't apply to nonexistent routes
        /**
         * POST /api/nginx/streams
         *
         * Create a new stream
         */
        .post((req, res, next) => {
                apiValidator({$ref: 'endpoints/streams#/links/1/schema'}, req.body)
                        .then((payload) => {
                                return internalStream.create(res.locals.access, payload);
                        })
                        .then((result) => {
                                res.status(201)
                                        .send(result);
                        })
                        .catch(next);
        });
```

Found the error message here:
```
/app/node_modules/ajv/lib/dot/errors.def:112:  oneOf:           "'should match exactly one schema in oneOf'",
/app/node_modules/ajv/lib/dotjs/oneOf.js:50:      out += ' , message: \'should match exactly one schema in oneOf\' ';
```

**Comment by j0hnnycache** [comment]
I'm seeing this issue too.
Using the npm docker v2.9.8 on unraid v6.9.2

**Comment by jc21** [comment]
This is fixed #1366 and available on the `github-develop` docker tag until officially released.
